Packs/WootCloud/pack_metadata.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "WootCloud",
+    "description": "WootCloud HyperContext Powered Security",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Luis Torres",
+    "url": "https://github.com/ArmatureSystems",
+    "email": "luis@armaturesystems.com",
+    "created": "2020-06-25T20:34:32Z",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": []
```

```suggestion
    "keywords": [],
    "githubUser": [users to be notified.]
```
Add to the githubUser all the users that need to be notified in case of a bug or question.